README.md

```diff
@@ -41,3 +41,5 @@ https://annu12340.github.io/Javascript-Notes/
 
 ### CSS Notes
 https://github.com/ResourceAggregator/Frontend-stuff/blob/master/css/css.md
+To know the difference between local and session storage visit the file(local vs session storage)  or this 
```

Comment:
Could you add this link under 'Good references' (section after Index>.. >advanced>**"Good references"** )
And it would be great if you could add a medium article or any other site as reference also (Something that explains the concept in a simpler manner)